Transforms/src/main/java/org/endeavourhealth/informationmanager/transforms/sources/OdsImporter.java

```diff
@@ -101,6 +105,10 @@ private void processHeaders() {
     private void processLine(TTDocument doc) {
         String odsCode = fieldByName("OrganisationId");
         String orgIri = IM.ORGANISATION_NAMESPACE +  UUID.randomUUID();
+        List<String> organisationList = List.of("RQX42", "8HL46", "5LA19", "RQM93", "RAX0A", "NV178", "RF4", "8HN02");
```

Comment:
Rather than having this list twice, is it possible to reference organisationMap.values() instead?